.changes/v0.13.6.md

```diff
@@ -0,0 +1,36 @@
+## v0.13.6 - 2024-10-24
+
+
+### 🔥 Breaking Changes
+- `dagger init` defaults to use `.dagger` folder during if current directory `.` is not empty by @rajatjindal in https://github.com/dagger/dagger/pull/8557
```

Comment:
Hmmm... this is actually not a breaking change. The new behavior is for new modules. Existing ones should be unaffected.

Reply:
Agreed, moved to "Changed".